common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -462,16 +463,7 @@ public static String frontStringAtLeast(final String str, final int size) {
     }
 
     public static boolean isBlank(String str) {
```

Comment:
How to delete this method and replace all with StringUtils.isBlank?

Reply:
Let's keep this UtilAll.isBlank , for there are more than 31 places using this method.

The current fix will keep the changes line at the minimum.